src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -290,8 +294,10 @@ private void updateCAType(Topic topic) {
             uploadCoreDeviceCAs(caCerts);
             updateCACertificateConfig(caCerts);
             updateCaPassphraseConfig(certificateManager.getCaPassPhrase());
+        } catch (CloudServiceInteractionException e) {
+            logger.atError().cause(e).log("Unable to upload core CA certs to the cloud. ");
         } catch (KeyStoreException | IOException | CertificateEncodingException | IllegalArgumentException
-                | CloudServiceInteractionException e) {
+                e) {
```

Comment:
who is calling this method? Is it safe to block the calling thread indefinitely?

Reply:
Does this log the failure twice? It's already logged below as part of `uploadCoreDeviceCAs`